### ISSUE TITLE
PDC gem must have a way to turn off ssl verification

### ISSUE THREAD

**Issue by sthaha** [product-definition-center/pdc-ruby-gem#4]
For testing pdc gem, there must be a way to turn off ssl verification

e.g.
```ruby
config.ssl_verify = false should result in faraday.new(... ssl: {verify: false}) 
```

**Comment by sthaha** [comment]
solved by #10

**Comment by sthaha** [comment]
SSL Verification is turned off for faraday but not when fetching token. Hence reopening.

**Comment by sthaha** [comment]
Fixed in PR #14
